Resources/translations/fieldedit.en.xlf

```diff
@@ -145,6 +145,18 @@
         <note>key: date.time.required</note>
         <jms:reference-file>./Resources/public/js/views/fields/ez-dateandtime-editview.js</jms:reference-file>
       </trans-unit>
+      <trans-unit id="a26a9e8cceea9e78f5ed30052cd1e0aa592498ca" resname="dateandtime.placeholder.date">
+        <source>YYYY-MM-DD</source>
+        <target>YYYY-MM-DD</target>
+        <note>key: dateandtime.placeholder.date</note>
+        <jms:reference-file>Resources/public/templates/fields/edit/dateandtime.hbt</jms:reference-file>
+      </trans-unit>
+      <trans-unit id="6cb8a97a772599f1562670c41d57373303d2c3e0" resname="dateandtime.placeholder.time">
+        <source>hh:mm:ss</source>
+        <target>hh:mm:ss</target>
```

Comment:
having `:ss` in the placeholder actually depends on the `useSeconds` field definition, so you need to different placeholder strings.